segment_executiontime.go

```diff
@@ -35,3 +48,115 @@ func (t *executiontime) init(props *properties, env environmentInfo) {
 	t.props = props
 	t.env = env
 }
+
+func (t *executiontime) formatDuration(ms int64, style DurationStyle) string {
+	switch style {
+	case AustinStyle:
+		return t.formatDurationAustin(ms)
+	case RoundrockStyle:
+		return t.formatDurationRoundrock(ms)
+	case DallasStyle:
+		return t.formatDurationDallas(ms)
+	case GalvestonStyle:
+		return t.formatDurationGalveston(ms)
+	case HoustonStyle:
+		return t.formatDurationHouston(ms)
+	case AmarilloStyle:
+		return t.formatDurationAmarillo(ms)
+	default:
+		return fmt.Sprintf("Style: %s is not available", style)
+	}
+}
+
+func (t *executiontime) formatDurationAustin(ms int64) string {
+	if ms < 1000 {
+		return fmt.Sprintf("%dms", ms%1000)
+	}
+
+	seconds := float64(ms%60000) / 1000
+	result := strconv.FormatFloat(seconds, 'f', -1, 64) + "s"
+
+	if ms >= 60000 {
```

Comment:
Looking at this, the `6000`, `3600000` etc numbers are recurring and can be considered as magic variables (you need to do the math). Can we replace these with a meaningful constant?

```go
const (
    second = 1000
    minute = 60000
    hour = 3600000
    ....
)
```

Reply:
done